csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -1927,82 +2054,94 @@ A CSAF content management system satisfies the "CSAF content management system"
 
 * When updating an exsting CSAF document:
   
-  * it prefills all fields which have be present in the existing CSAF document
-  * it adds a new item in `/document/tracking/revision_history[]`
-  * it updates the following fields with the values given below or based on the templates from configuration:
-  * `/document/csaf_version` with the value `2.0`
-  * `/document/language`
-  * `/document/notes`
-    * `legal_disclaimer` (Terms of use from the configuration)
-    * `general` (General Security recommendations from the configuration)
-  * `/document/tracking/current_release_date` with the current date
-  * `/document/tracking/generator` and children
-  * the new item in `/document/tracking/revision_history[]`
-    * `date` with the current date
-    * `number` (based on the templates from configuration; default: latest_patch_version + 1)
-  * `/document/tracking/status` with `draft`
-  * `/document/tracking/version` with the value of `number` the latest `/document/tracking/revision_history[]` element
-  * `/document/publisher` and children
+  * prefills all fields which have be present in the existing CSAF document
+  * adds a new item in `/document/tracking/revision_history[]`
+  * updates the following fields with the values given below or based on the templates from configuration:
+      * `/document/csaf_version` with the value `2.0`
+      * `/document/language`
+      * `/document/notes`
+        * `legal_disclaimer` (Terms of use from the configuration)
+        * `general` (General Security recommendations from the configuration)
+      * `/document/tracking/current_release_date` with the current date
+      * `/document/tracking/generator` and children
+      * the new item in `/document/tracking/revision_history[]`
+        * `date` with the current date
+        * `number` (based on the templates from configuration; default: latest_patch_version + 1)
+      * `/document/tracking/status` with `draft`
+      * `/document/tracking/version` with the value of `number` the latest `/document/tracking/revision_history[]` element
+      * `/document/publisher` and children
+
+### 5.1.7 Conformance Clause 7: CSAF post-processor
+
+A CSAF post-processor satisfies the "CSAF post-processor" conformance profile if the post-processor:
+
+* satisfies the "CSAF consumer" conformance profile.
+* satisfies the "CSAF producer" conformance profile.
+* additionally satisfies those normative requirements in section 3 that are designated as applying to post-processors.
```

Comment:
Such normative requirements do not exist yet in section 3 (if I remember correctly). Do we want to introduce them?

Reply:
I guess the best would be to refer to section 5 as we have done in the other cases.